frontend/src/employee-frontend/components/unit/unit-details/UnitEditor.tsx

```diff
@@ -81,6 +87,7 @@ interface FormData {
   ophUnitOid: string
   ophOrganizerOid: string
   operationDays: DayOfWeek[]
+  operationTimes: EditableTimeRange[]
```

Comment:
We now have two sources of truth in this form. Is there really a need to retain the old `operationDays` array if `operationTimes` is an array with guaranteed 7 nullable elements?

Reply:
Removed and refactored to dynamically produce days from times.